teamserver/pkg/comms/actionrequest.go

```diff
@@ -19,11 +20,21 @@ func ActionRequestHandler(packet Packet) ([]Packet, error) {
 		return nil, fmt.Errorf("not registered")
 	}
 
+	// Update that the implant called back
 	_, err := d.UpdateCallBackTime(packet.Implant.UUID, time.Now().Format(crud.TimeStamp))
 	if err != nil {
 		return nil, fmt.Errorf("not registered")
 	}
 
+	// -------------- This is for the pwnboard plugin ------------------
+	// -------- This is not needed for core functionality --------------
+	if plugins.PWNBOARD_ENABLED {
```

Comment:
Might wanna rework how this is done. Maybe have a plugin manager that knows when plugins execute and call that instead. Then it calls the enabled plugins. I only say this cause when more plugins start being developed this is going to get messy the way it is.

Reply:
For example at set points in the code (specifically labeled), we make a call to the plugin manager for plugin callbacks. If a plugin is set to do stuff at that set time the plugin manager runs the plugin function. Makes it so that the core framework is not modified every time a plugin is created.

Reply:
This is a good point, lets talk about this and I'll comment the results.

Reply:
There will be a plug in manager that will help solve this later. This will allow for others to create their own plugins and not have to modify different aspects of the code.